dataset/synthetic/01_random_replication/access_log.txt

```diff
@@ -0,0 +1,60 @@
+1	1	5.5.5.1	null	4.4.4.1	WRITE	201	88
+2	2	5.5.5.2	null	4.4.4.1	WRITE	201	184
+3	3	5.5.5.3	null	4.4.4.1	WRITE	201	345
+4	4	5.5.5.4	null	4.4.4.1	WRITE	201	316
+5	5	5.5.5.5	null	4.4.4.1	WRITE	201	300
+6	6	5.5.5.6	null	4.4.4.2	WRITE	201	178
+7	7	5.5.5.7	null	4.4.4.2	WRITE	201	170
+8	8	5.5.5.8	null	4.4.4.2	WRITE	201	112
+9	9	5.5.5.9	null	4.4.4.2	WRITE	201	45
+10	10	5.5.5.10	null	4.4.4.2	WRITE	201	215
+11	1	5.5.5.6	null	4.4.4.2	READ	200	null
```

Comment:
now i see why this is weird to actually have response codes in the dataset, because realistically, this is a `302` response, right?

don't think it's a big deal because we'll just ignore it, but just a thought